packages/record-data/addon/-private/relationships/state/relationship.ts

```diff
@@ -74,6 +75,11 @@ export default class Relationship {
     this.meta = null;
     this.__inverseMeta = undefined;
 
+    this.links = undefined;
```

Comment:
Do you think we need this one (or the one above it)?  I would assume we get that anyways for free without the explicit declaration?

Reply:
Yes. It shapes the object so that it doesn't transition later when we do set it.